sqlite/src/func.c

```diff
@@ -971,6 +973,26 @@ static void comdb2StartTimeFunc(
   sqlite3_result_datetime(context, &dt, NULL);
 }
 
+static void md5Func(
+  sqlite3_context *context,
+  int argc,
+  sqlite3_value **argv
+  ){
+    const unsigned char *s;
+    MD5Context ctx = {0};
+    char checksum[16];
+    char out[33];
+
+    if (sqlite3_value_type(argv[0]) != SQLITE_TEXT)
```

Comment:
It would also be nice to be able to perform the md5sum() function on a blob value.